Source/WebKit/WebProcess/com.apple.WebProcess.sb.in

```diff
@@ -448,7 +448,7 @@
         "kern.osversion"
         "kern.safeboot"
         "kern.version"
-        "kern.willshutdown" ;; Silence after <rdar://125745970> is resolved.
+        "kern.willshutdown" ;; <rdar://122511261>
```

Comment:
If possible, I think it would be good to silence this in the WebContent processes.